json/Card/my-ancestors-strength.json

```diff
@@ -0,0 +1,30 @@
+[
+    {
+        "clan": "phoenix",
+        "cost": 1,
+        "deck_limit": 3,
+        "element": null,
+        "fate": null,
+        "glory": null,
+        "honor": null,
+        "id": "my-ancestors-strength",
```

Comment:
actually that card already exists too, and it's in the file my-ancestor-s-strength.json

Reply:
so you should remove this file